### PR TITLE
New version 1.0.0

### PR DESCRIPTION
**Pull request by noviluni**
New version 1.0.0

### CODE REVIEW

File: HISTORY.rst
Comment (Gallaecio) [inline HISTORY.rst:13]:
```suggestion
* Now ``dateparser.parse()`` is deterministic and doesn't try previous locales. On the other hand, ``DateDataParser.get_date_data()`` doesn't try the previous locales by default (see #781)
```

Comment (Gallaecio) [reply HISTORY.rst:13]: From reading this, I’m a bit confused because it says “on the other hand” but seems to state that `DateDataParser.get_date_data()` behaves the same as `parse` (by default).

File: HISTORY.rst
Comment (Gallaecio) [inline HISTORY.rst:19]:
```suggestion
* Changes in ``dateparser.utils``: ``normalize_unicode()`` doesn't accept ``bytes`` as input and ``convert_to_unicode`` has been deprecated.
```

File: HISTORY.rst
Comment (Gallaecio) [inline HISTORY.rst:25]:
```suggestion
* Add support for "decade" (see #762)
```

File: HISTORY.rst
Comment (Gallaecio) [inline HISTORY.rst:24]:
```suggestion
* Detect hours separated with a period/dot (see #741)
```

File: HISTORY.rst
Comment (Gallaecio) [inline HISTORY.rst:31]:
```suggestion
* Improve the README and docs (see #779)
```

File: HISTORY.rst
Comment (Gallaecio) [inline HISTORY.rst:34]:
```suggestion
* Parse 24hr time formats with meridian info (see #634)
```

or

```suggestion
* Parse the 24hr time format with meridian info (see #634)
```

File: HISTORY.rst
Comment (noviluni) [reply HISTORY.rst:13]: @Gallaecio 

"[...] is deterministic and doesn't try previous locales, while ``DateDataParser.get_date_data()`` doesn't  try the previous locales by default"

would sound better this?

I'm not sure how to write this... What I want to say is:
* `dateparser.parse()` is now deterministic and it doesn't try the previously used locales.
* `DateDataParser.get_date_data()` by default doesn't try the previously used locales.

Could you help me to express it better? :sweat_smile:

Comment (Gallaecio) [reply HISTORY.rst:13]: Maybe just “also” instead of “in the other hand” would make things more clear to me. Let me look again into the actual changes and get back to you.

**Review by Gallaecio — APPROVED** [review]

**Comment by noviluni** [comment]
It's ready for review. I will just update the date when we merge the last two missing PRS (presumably tomorrow):
* https://github.com/scrapinghub/dateparser/pull/823 (waiting for CI to finish)
* https://github.com/scrapinghub/dateparser/pull/778 (waiting for a second review from @Gallaecio

**Pull request merged**
